scripts/Reflectometry/isis_reflectometry/procedures.py

```diff
@@ -5,7 +5,7 @@
 #   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
 # SPDX - License - Identifier: GPL - 3.0 +
 # pylint: disable=too-many-lines, invalid-name, too-many-arguments, too-many-branches, too-many-locals
-
+# ruff: noqa: F403   # Allow wild imports
```

Comment:
Is this one ignored because it probably doesn't run anyway? I see some `mantidplot`s in there

Reply:
Yes I ignored this one because it looks like its mostly unused, and also it had an insane number of wild imports so it would take a long time to fix it

Reply:
Understandable 👍 we should ask LSS if it's still needed